easybuild/tools/utilities.py

```diff
@@ -223,6 +223,25 @@ def nub(list_):
     return [x for x in list_ if x not in seen and not seen_add(x)]
 
 
+def unique_ordered_append(base, extra):
+    """Append elements of extra list to another base list keeping order and without duplicates"""
+    if extra and isinstance(extra, str):
+        extra = [extra]
+
+    try:
+        base.extend(extra)
+    except TypeError as err:
+        err_msg = f"_unique_ordered_list_append: given extra list is not iterable: {extra}"
+        raise EasyBuildError(err_msg) from err
```

Comment:
The backtrace should already contain the origin function avoiding using the wrong name here ;-)
```suggestion
        raise EasyBuildError( f"given extra list is not iterable: {extra}") from err
```

Reply:
right :slightly_smiling_face: